crates/biome_css_analyze/src/keywords.rs

```diff
@@ -855,13 +855,73 @@ pub const OTHER_PSEUDO_ELEMENTS: [&str; 18] = [
     "view-transition-old",
 ];
 
-pub const VENDER_PREFIXES: [&str; 4] = ["-webkit-", "-moz-", "-ms-", "-o-"];
+pub const MEDIA_FEATURE_NAMES: [&str; 57] = [
+    "any-hover",
+    "any-pointer",
+    "aspect-ratio",
+    "color",
+    "color-gamut",
+    "color-index",
+    "device-aspect-ratio",
+    "device-height",
+    "device-width",
+    "display-mode",
+    "dynamic-range",
+    "environment-blending",
+    "forced-colors",
+    "grid",
+    "height",
+    "horizontal-viewport-segments",
+    "hover",
+    "inverted-colors",
+    "max-aspect-ratio",
+    "max-color",
+    "max-color-index",
+    "max-device-aspect-ratio",
+    "max-device-height",
+    "max-device-width",
+    "max-height",
+    "max-monochrome",
+    "max-resolution",
+    "max-width",
+    "min-aspect-ratio",
+    "min-color",
+    "min-color-index",
+    "min-device-aspect-ratio",
+    "min-device-height",
+    "min-device-width",
+    "min-height",
+    "min-monochrome",
+    "min-resolution",
+    "min-width",
+    "monochrome",
+    "nav-controls",
+    "orientation",
+    "overflow-block",
+    "overflow-inline",
+    "pointer",
+    "prefers-color-scheme",
+    "prefers-contrast",
+    "prefers-reduced-motion",
```

Comment:
`prefers-reduced-data` is missing.

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-data

Reply:
This feature is described as following,

> Note: This feature is not supported by any user agent and its specifics are subject to change.

so, it should not be included.

Reply:
No it's supported behind the `enable-experimental-web-platform-features` flag since version 85.

see https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-data#browser_compatibility

Reply:
I'm sorry but I misunderstand, I fixed.
Thanks 🙏